python/cudf/cudf/utils/dtypes.py

```diff
@@ -651,7 +656,59 @@ def find_common_type(dtypes):
         dtypes = dtypes - td_dtypes
         dtypes.add(np.result_type(*td_dtypes))
 
-    return np.find_common_type(list(dtypes), [])
+    common_dtype = np.find_common_type(list(dtypes), [])
+    if common_dtype == np.dtype("float16"):
+        # cuDF does not support float16 dtype
+        return np.dtype("float32")
+    else:
+        return common_dtype
+
+
+def _can_cast(from_dtype, to_dtype):
```

Comment:
Is this meant to be different from https://github.com/rapidsai/cudf/blob/branch-0.18/python/cudf/cudf/core/column/numerical.py#L562? Now we have two methods that do somewhat similar things where it'd be nice to have just one if possible.

Reply:
Synced offline and these are not the same